File: jats-demo-model/src/main/java/com/github/pepe79/jats/demo/model/Quantity.java
package com.github.pepe79.jats.demo.model;

public class Quantity
{
	private Uom uom;

	private double quantity;

	public Uom getUom()
	{
		return uom;
	}

	public void setUom(Uom uom)
	{
		this.uom = uom;
	}

	public double getQuantity()
	{
		return quantity;
	}

	public void setQuantity(double quantity)
	{
		this.quantity = quantity;
	}

	public String getDisplayQuantity()
	{
		return quantity + " " + uom.getCode();
	}

}

File: jats-js/src/test/ts/demo/model/Quantity.js
var Quantity = (function () {
    function Quantity() {
    }
    Quantity.prototype.getUom = function () {
        return this.uom;
    };
    Quantity.prototype.setUom = function (uom) {
        this.uom = uom;
    };

    Quantity.prototype.getQuantity = function () {
        return this.quantity;
    };
    Quantity.prototype.setQuantity = function (quantity) {
        this.quantity = quantity;
    };

    Quantity.prototype.getDisplayQuantity = function () {
        return this.displayQuantity;
    };
    return Quantity;
})();
